be/src/formats/orc/orc_chunk_reader.cpp

```diff
@@ -1896,7 +1895,11 @@ static orc::Literal translate_to_orc_literal(Expr* lit, orc::PredicateDataType p
     }
 
     auto* vlit = down_cast<VectorizedLiteral*>(lit);
-    ColumnPtr ptr = vlit->evaluate(nullptr, nullptr);
+    auto maybe_ptr = vlit->evaluate_checked(nullptr, nullptr);
+    if (!maybe_ptr.ok()) {
+        return {pred_type};
+    }
```

Comment:
Is it ok to swallow error status here?

Reply:
I should refactor this function to propagate error here.